-------------------------------------------------------------------------------
Test set: com.manus.patrimonio.exception.RecursoNaoEncontradoExceptionTest
-------------------------------------------------------------------------------
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.003 s -- in com.manus.patrimonio.exception.RecursoNaoEncontradoExceptionTest
